GKE-runner/variables.tf

```diff
@@ -0,0 +1,376 @@
+  type        = string
+  default     = "gitlab-runner"
+}
+
+variable "additional_node_service_account_roles" {
+  description = "additional roles to grant the service account"
+  type        = list(any)
+  default     = []
+}
+
+variable "runner_node_pool_name" {
+  description = "name of the runner node pool"
+  type        = string
+  default     = null
+}
+
+variable "initial_node_count" {
+  description = "initial number of nodes that the node pool creates"
+  type        = number
+  default     = 0
+}
+
+variable "runner_node_pool_min_node_count" {
+  description = "the minimum number of nodes that can be present in the node pool (autoscaling controls)"
+  type        = number
+  default     = 0
+}
+
+variable "runner_node_pool_max_node_count" {
+  description = "the maximum number of nodes that can be present in the node pool (autoscaling controls)"
+  type        = number
+  default     = 5
+}
+
+variable "runner_node_pool_machine_type" {
+  description = "type of compute machine used for the nodes in the runner node pool"
+  type        = string
+  default     = "n1-standard-2"
+}
+
+variable "runner_node_pool_node_labels" {
+  description = "labels for nodes in the runner node pool"
+  type        = map(any)
+  default = {
+    "role" = "gitlab-runner"
+  }
+}
+
+variable "runner_node_pool_node_taints" {
+  description = "taints to be applied to the nodes in the runner node pool"
+  type        = list(map(string))
+  default = [{
+    effect = "NO_SCHEDULE"
+    key    = "role"
+    value  = "gitlab-ci"
+  }]
+}
+
+# variable "node_is_preemptible" {
+#   default     = false
+#   type        = bool
+#   description = "A boolean that represents whether or not the underlying node VMs are preemptible"
+# }
+
+variable "runner_node_pool_oauth_scopes" {
+  description = "(Optional) Scopes that are used by NAP when creating node pools."
+  default     = ["https://www.googleapis.com/auth/cloud-platform"]
+  type        = list(string)
+}
+
+variable "cache_location" {
+  description = "location of the cache bucket"
+  type        = string
+  # default     = null
+}
+
+variable "cache_storage_class" {
+  description = "The cache storage class"
+  type        = string
+  default     = "STANDARD"
+}
+variable "cache_labels" {
+  description = "The cache storage class"
+  type        = map(string)
+  default = {
+    "role" = "gitlab-runner-cache"
+  }
+}
+
+variable "cache_expiration_days" {
+  description = "Number of days before cache objects expires."
+  type        = number
+  default     = 2
+}
+
+variable "cache_bucket_versioning" {
+  description = "Boolean used to enable versioning on the cache bucket, false by default."
+  type        = bool
+  default     = false
+}
+
+variable "runner_release_name" {
+  description = "helm release name"
+  type        = string
+  default     = null
+}
+
+variable "cache_path" {
+  description = "path to append to the bucket url"
+  type        = string
+  default     = "runner"
```

Comment:
I'm not sure we need this. Set the default to `""`. This is a dedicated bucket for caching, there's no need adding a path to it